src/Pages/Store.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Route, Link, Routes } from 'react-router-dom';
+import CoffeeStoreContainer from '../Components/CoffeeStoreContainer';
+import BrewerStoreContainer from '../Components/BrewerStoreContainer';
+import { useIntl } from 'react-intl';
+
+interface StoreProps {}
```

Comment:
It is not necessary to have empty interfaces

Reply:
[85337de](https://github.com/Kentico/kontent-sample-app-react/pull/206/commits/85337de168d15d545c42aa5a5e69b45bc9ec5a11)